steam/ext/dota2/state.py

```diff
@@ -3,7 +3,6 @@
 from __future__ import annotations
 
 import asyncio
-from functools import partial
```

Comment:
I didn't realise that this was in a loop and using lambdas leads to a bug where the check is only getting the last start match id (I think). So this might need reverting